.github/utilities/md-spellcheck/spellcheck.yml

```
@@ -0,0 +1,24 @@
+matrix:
+  - name: Markdown
+    aspell:
+      lang: en
+    dictionary:
+      wordlists:
+      - wordlist.txt
+      encoding: utf-8
+    pipeline:
+      - pyspelling.filters.markdown:
+          markdown_extensions:
+          - markdown.extensions.extra:
+      - pyspelling.filters.html:
+          comments: true
+          attributes:
+          - title
+          - alt
+          ignores:
+          - ':matches(code, pre)'
+          - 'code'
+          - 'pre'
+    sources:
+    - '../../../**/*.md'
```

This can be simplified

```suggestion
    - '/**/*.md'
```

---

Perhaps - '${GITHUB_WORKSPACE}/**/*.md'